src/deploy/functions/tasks.ts

```diff
@@ -116,6 +116,16 @@ export function createFunctionTask(
         params.errorHandler.record("warning", fnName, "make public", err.message);
       }
     }
+    if ("concurrency" in fn) {
+      if (fn.platform === "gcfv1") {
+        logger.debug("Cannot set concurrency for a v1 function");
+      } else {
+        await setConcurrency(
+          (cloudFunction as gcfV2.CloudFunction).serviceConfig.service!,
+          fn.concurrency || 1
```

Comment:
We should default to `n > 1` for creates on GCFv2 platform. My suggestion would be 50 or 80.